.gitmodules

```diff
@@ -10,3 +10,9 @@
 [submodule "lib/openzeppelin-contracts"]
 	path = lib/openzeppelin-contracts
 	url = https://github.com/openzeppelin/openzeppelin-contracts
+[submodule "lib/protocol-v2"]
```

Comment:
We don't seem to use this anywhere, at least not currently.

Reply:
IMO we should remove this if we aren't currently using it

Reply:
Done in 8a07568